### ISSUE TITLE
Regarding image normalization during img_proj in DnceLatentProj

### ISSUE THREAD

**Issue by andrewwwj** [Dstate/LBP#5]
Hi, thanks for sharing this amazing work and I'm pretty impressed by your work.

During the reproduction, I have a question about the normalization during img_proj:
https://github.com/Dstate/LBP/blob/f97c5fc9372064eadd531c477e50ed2e05d6a86d/models/MidPlanner.py#L64

Both camera views (main view and wrist view) are utilized in vision_encoder in LBP.py but they are normalized with the same parameters, using image0 statistics:
img_mean=torch.tensor(data['img_latents_dnce']['image0']['mean'])
img_std=torch.tensor(data['img_latents_dnce']['image0']['std'])
although each view's mean and std are stored in assets/libero.pkl.

While it seems this doesn't significantly affect the performance, as I was able to reproduce your results, is there a specific reason why you chose to use the main view's statistics only?

**Comment by Dstate** [comment]
Hi, thank you for your interest in our work!
Actually, while both views are used in the policy, our planning is conducted only on main-view images: https://github.com/Dstate/LBP/blob/f97c5fc9372064eadd531c477e50ed2e05d6a86d/models/MidPlanner.py#L97

The reason is that the wrist view provide limited information beyond gripper status, and its rapid changes make it less suitable for planning.
